backend/migrator/migration/dev/LATEST.sql

```diff
@@ -332,7 +332,8 @@ CREATE TABLE instance (
     external_link TEXT NOT NULL DEFAULT '',
     resource_id TEXT NOT NULL,
     -- activation should set to be TRUE if users assign license to this instance.
-    activation BOOLEAN NOT NULL DEFAULT false
+    activation BOOLEAN NOT NULL DEFAULT false,
+    option JSONB NOT NULL DEFAULT '{}'
```

Comment:
Let's call it options since we already have something called

message DataSourceOptions